project/Dependencies.scala

```diff
@@ -24,7 +24,7 @@ object Dependencies {
     val akkaPersistenceJdbc = "5.2.0"
     val r2dbc = "1.0.0.RELEASE"
     // FIXME non-milestone
-    val akkaPersistenceR2dbc = "1.1.0-M5"
+    val akkaPersistenceR2dbc = "1.1.0-M8"
```

Comment:
strange that this is shown in the diff, it's already M8 in main

Reply:
My fork is not up-to date, but still it should not.

Reply:
A rebase on the latest version resolved it.